tasks/notify.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: notify Hipchat
+  hipchat:
+    api: https://api.hipchat.com/v2/
+    token: "{{ hitchat_access_token }}"
+    room: "{{ hitchat_room }}"
+    msg: "/code {{ notify_message }}"
```

Comment:
It's interesting to change the color when is successful applied or not.

Reply:
Done 👍